src/main/test/LruCacheTests.cpp

```diff
@@ -1,132 +0,0 @@
-// Copyright 2014 Stellar Development Foundation and contributors. Licensed
-// under the Apache License, Version 2.0. See the COPYING file at the root
-// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
-
-#include "lib/catch.hpp"
-#include "lib/util/lrucache.hpp"
-
-namespace stellar
-{
-
-using IntCache = cache::lru_cache<int, int>;
```

Comment:
why are those tests gone?

Reply:
should be moved to the util/test folder instead

Reply:
Both Lru Cache and random eviction tests are covered by template tests in `CacheTests.cpp`. That file is already in `util/test`.

Reply:
fancy!